standards-catalogue/Rakefile

```diff
@@ -26,11 +26,13 @@ def is_valid_schema(schema, to_validate)
   false
 end
 
-def is_data_file_schema_compliant(schema, to_validate)
+def is_data_file_schema_compliant(schema, to_validate, key_regex)
   schema = schema(schema)
   objects = YAML.load_file(to_validate)
-  errors = objects.map do |_, object|
-    is_valid_schema(schema, object)
+  errors = objects.map do |key, object|
+    key_valid = key_regex =~ key
+    puts "#{to_validate} failed validation: `#{key}` did not match the expected pattern (`#{key_regex}`)" unless key_valid
```

Comment:
wondering if this error message would be hard to follow for people adding a new standard - could it be `#{key} must be all lower case letters` etc, or is that over complicating it?